### DISCUSSION TITLE
Epic gems from Hyjal purchasable with honor points

### DISCUSSION THREAD

**Discussion by Eledor-collab** [chromiecraft/chromiecraft#7109] [Suggestions]
Increase PvP activity in the server by allowing epic gems to be purchased with honor points - similar to what happened on the first months of arena season 2 allowing set s1 to be purchasable with Honor Points.

Reasons to consider this:

1. It is almost impossible to obtain epic gems for people that dedicate most of  their time doing PvP.
2. PvP activity is relatively low at the moment.
3. Epic gems may be the only way to obtain relevant stats for many specs and classes, i.e., haste for spriest/eleshams/mages.
4. Impact on server economy can be easily controlled, either by making them BoP or deciding to make them very pricey like 75k each - and tbh increasing supply will make us have healthier prices at AH for those gems.
5. Make ppl who have made a home in the server and love PvP feel important. 
6. why not? :P

**Comment by diminsionz** [comment]
just make them bind on pick up
